judge/admin/profile.py

```diff
@@ -54,18 +54,20 @@ def has_add_permission(self, request):
 
 
 class ProfileAdmin(NoBatchDeleteMixin, VersionAdmin):
-    form = ProfileForm
     fieldsets = (
         (None, {'fields': ('user', 'display_rank')}),
         (_('User Settings'), {'fields': ('organizations', 'timezone', 'language', 'ace_theme', 'math_engine')}),
         (_('Administration'), {'fields': ('is_external_user', 'mute', 'is_unlisted', 'is_totp_enabled',
-                                          'last_access', 'ip', 'current_contest', 'notes')}),
+                                          'last_access', 'ip', 'current_contest', 'notes',
+                                          'username_display_override')}),
         (_('Text Fields'), {'fields': ('about', 'user_script')}),
     )
-    list_display = ('user', 'full_name', 'email', 'is_totp_enabled', 'is_external_user',
+    readonly_fields = ('user',)
+    list_display = ('admin_user_admin', 'email', 'is_totp_enabled', 'timezone_full',
+                    'full_name', 'is_external_user',
                     'date_joined', 'last_access', 'ip', 'show_public')
     ordering = ('user__username',)
-    search_fields = ('user__username', 'user__first_name', 'user__last_name', 'ip', 'user__email')
```

Comment:
Any reason to remove these?

Reply:
No specific reason other than that (I think) username display overrides were added and names don't make too much sense anymore (also I think users can't set them).

Reply:
Names are set by users on signup to the site (MCPT specific, not found on DMOJ), searching by name is useful for us.